modules/nf-core/deepcell/mesmer/main.nf

```diff
@@ -0,0 +1,40 @@
+process DEEPCELL_MESMER {
+    tag "$meta.id"
+    label 'process_medium'
```

Comment:
there are no docs to say it can use task.cpus, does it run on multiple threads? Otherwise process_single might also be an option

Reply:
I think it simply uses all available threads by default. I can change it to process_single. I believe the reason they chose medium was mostly RAM hungriness.

Reply:
ah then it's fine. was just wondering if you would request 12 and then only use 1.